src/freenet/node/useralerts/UserAlert.java

```diff
@@ -4,6 +4,7 @@
 package freenet.node.useralerts;
 
 import freenet.clients.fcp.FCPMessage;
+import freenet.node.PeerNode;
```

Comment:
I wonder what this import is about, as apparently no other change in this file even so much as thinks of mentioning `PeerNode`… 😁